Archive/eventarchive.html

```diff
@@ -64,8 +64,8 @@ <h1 class = "text-center"><strong>Events Archive</strong></h1>
             </div>
             <div class="row mt-4 mb-4 mx-4 justify-content-left">
                 <div class="col-lg-4 my-2">
-                    <div class="card">
-                        <img class="card-img-top" src="Images/CurrentEvents/CSharp Basics Workshop Banner.png" alt="C# Workshop Banner">
+                    <div class="card card-shadow">
+                        <img class="card-img-top" src="PastEvents/CSharp Basics Workshop Banner.png" alt="C# Workshop Banner">

```

Comment:
I think it would be nice to have a top-level `Images/` folder, but this is fine if it's how it is now